scripts/opencontrail-kubelet/opencontrail_kubelet/plugin.py

```diff
@@ -98,8 +107,8 @@ def getDockerPod(docker_id):
 def getPodInfo(podName):
     kubeapi = kubelet_get_api()
 
-    data = Shell.run('kubectl --server=%s get -o json pod %s' % (
-        kubeapi, podName))
+    data = Shell.run('kubectl --server=%s:7080 get -o json pod %s' % (
```

Comment:
Hi Pedro,

Pull request has been updated with your comments. I have also added command
line options to the install script.

Thanks,
Ananth

On Mon, Jul 6, 2015 at 3:10 PM, Pedro Marques <notifications@github.com>
wrote:

> In scripts/opencontrail-kubelet/opencontrail_kubelet/plugin.py
> <https://github.com/Juniper/contrail-kubernetes/pull/13#discussion_r33988396>
> :
>
> > @@ -98,8 +107,8 @@ def getDockerPod(docker_id):
> >  def getPodInfo(podName):
> >      kubeapi = kubelet_get_api()
> >
> > -    data = Shell.run('kubectl --server=%s get -o json pod %s' % (
> > -        kubeapi, podName))
> > +    data = Shell.run('kubectl --server=%s:7080 get -o json pod %s' % (
>
> Why use a non-default port ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Juniper/contrail-kubernetes/pull/13/files#r33988396>.
>